jsons/parameters.json

```diff
@@ -15897,5 +16149,840 @@
+          "gluc_bicarb"
+        ],
+        [
+          "iodosobenzoate",
+          "iodosobenzoate"
+        ],
+        [
+          "lysc",
+          "lysc"
+        ],
+        [
+          "lysc_gluc",
+          "lysc_gluc"
+        ],
+        [
+          "lysc_p",
+          "lysc_p"
+        ],
+        [
+          "lysn",
+          "lysn"
+        ],
+        [
+          "lysn_promisc",
+          "lysn_promisc"
+        ],
+        [
+          "pepsina",
+          "pepsina"
+        ],
+        [
+          "protein_endopeptidase",
+          "protein_endopeptidase"
+        ],
+        [
+          "staph_protease",
+          "staph_protease"
+        ],
+        [
+          "trypsin",
+          "trypsin"
+        ],
+        [
+          "trypsin_cnbr",
+          "trypsin_cnbr"
+        ],
+        [
+          "trypsin_gluc",
+          "trypsin_gluc"
+        ],
+        [
+          "trypsin_p",
+          "trypsin_p"
+        ]
+      ]
+    },
+    "value_type": "str"
+  },
+  {
+    "_id": 391,
+    "default_value": true,
+    "description": "A boolean flag which defines if isoleucin and leucin should be treated the same in terms of peptide identification (default is true).",
+    "key_translations": {
+      "comet_style_1": "equal_I_and_L",
+      "ursgal_style_1": "equal_isoleucin_leucin"
+    },
+    "name": "equal_isoleucin_leucin",
+    "tag": [
+      "output"
+    ],
+    "triggers_rerun": true,
+    "value_translations": {
+      "comet_style_1": [
+        [
+          true,
+          1
+        ],
+        [
+          false,
+          0
+        ]
+      ],
+      "ursgal_style_1": [
+        [
+          true,
+          true
+        ],
+        [
+          false,
+          false
+        ]
+      ]
+    },
+    "value_type": "bool"
+  },
+  {
+    "_id": 392,
+    "default_value": "",
+    "description": "This parameter controls whether the output text file, as controlled by the parameter output_txtfile, will have a default \".txt\" file extension or a custom extension specified by this parameter.",
+    "key_translations": {
+      "comet_style_1": "text_file_extension",
+      "ursgal_style_1": "text_file_extension"
+    },
+    "name": "text_file_extension",
+    "tag": [
+      "output"
+    ],
+    "triggers_rerun": false,
+    "value_translations": {},
+    "value_type": "str"
+  },
+  {
+    "_id": 393,
+    "default_value": 0,
+    "description": "This parameter controls how the peptide_mass_tolerance parameter is applied. That tolerance can be applied to the singly charged peptide mass (default=0) or it can be applied to the precursor m/z (1).",
+    "key_translations": {
+      "comet_style_1": "precursor_tolerance_type",
+      "ursgal_style_1": "precursor_tolerance_type"
+    },
+    "name": "precursor_tolerance_type",
+    "tag": [],
+    "triggers_rerun": true,
+    "value_translations": {},
+    "value_type": "int"
+  },
+  {
+    "_id": 394,
+    "default_value": 0,
```

Comment:
can only be 0 or 1 ... [link](http://comet-ms.sourceforge.net/parameters/parameters_201801/theoretical_fragment_ions.php)

Reply:
It is true that it can only be 0/1. However, I agree with Stefan, that the 0/1 are not values here but rather representation on how the parameter should be treated. Hence I changed the param and included the proper description and value translation :)